src/lib/index.ts

```diff
@@ -45,8 +45,12 @@ class AssetEntry {
         }
     }
     private applyScript(tag: HTMLScriptElement) {
+        const start = /<script>/gi;
+        const end = /<\/script>gi;
```

Comment:
missing `/` at the tail?

Reply:
yep - was retyping the fix from my work computer and bungled it.... fixed in latest push